tracing-subscriber/src/subscribe/mod.rs

```diff
@@ -417,6 +417,28 @@
 //! [`Interest::never()`] from its [`register_callsite`] method, filter
 //! evaluation will short-circuit and the span or event will be disabled.
 //!
+//! ### Enabling interest
```

Comment:
nit: generally both words of section headings are capitalized
```suggestion
//! ### Enabling Interest
```